core/src/avm2/globals/flash/display3D/textures/CubeTexture.as

```diff
@@ -1,6 +1,9 @@
 package flash.display3D.textures {
     import flash.display.BitmapData;
+    import flash.utils.ByteArray;
     public final class CubeTexture extends TextureBase {
-        public native function uploadFromBitmapData(source:BitmapData, side:uint, miplevel:uint = 0):void
+        public native function uploadFromBitmapData(source:BitmapData, side:uint, miplevel:uint = 0):void;
```

Comment:
While I'm not complaining about this stub, I don't think it was added intentionally.